docs/blameable.md

```diff
@@ -54,7 +54,9 @@ var_dump($updatedBy);
 By default, no user entity is provided. You need to specify the User class with a new parameter in your config:
 
 ```yaml
-# services.yaml
-parameters:
-    doctrine_behaviors_blameable_user_entity: App\Entity\User
+# doctrine_behaviors.yaml
+doctrine_behaviors:
```

Comment:
What about this?
```suggestion
# knp_doctrine_behaviors.yaml
knp_doctrine_behaviors:
```

KnpLabs's packages like [KnpSnappyBundle](https://github.com/KnpLabs/KnpSnappyBundle#configuration) or [KnpPaginatorBundle](https://github.com/KnpLabs/KnpPaginatorBundle#configuration-example) prefix their config file and root node's name by `knp_`

Reply:
Also I think it requires to update the DoctrineBehaviors bundle name to  `KnpDoctrineBehaviorsBundle` I think, see https://symfony.com/doc/current/bundles/configuration.html#using-the-bundle-extension

Reply:
That's why I let doctrine_behaviors (I prefer prefixing with knp too). 
So I create a new Bundle, deprecate the previous one and force the name.

Reply:
I think `octrine_behaviors` is unique enough now. No need to change it